basculehttp/log.go

```diff
@@ -69,11 +72,36 @@ func sanitizeHeaders(headers http.Header) (filtered http.Header) {
 func SetLogger(logger *zap.Logger) alice.Constructor {
 	return func(delegate http.Handler) http.Handler {
 		return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-			r = r.WithContext(sallust.With(r.Context(),
-				logger.With(
-					zap.Reflect("requestHeaders", sanitizeHeaders(r.Header)), //lgtm [go/clear-text-logging]
-					zap.String("requestURL", r.URL.EscapedPath()),
-					zap.String("method", r.Method))))
+			var tid string
+			if tid = r.Header.Get(candlelight.HeaderWPATIDKeyName); tid == "" {
```

Comment:
This is technically fine, but reads a little strange...
```suggestion
			tid := r.Header.Get(candlelight.HeaderWPATIDKeyName)
			if tid == "" {
```